uint64      time_start_ns
uint32      duration_ms
string[]    mac_address
int16[]     freq
RssDatum[]  data
